ArduCopter/RC_Channel.cpp

```diff
@@ -580,14 +580,39 @@ void Copter::save_trim()
 
 // auto_trim - slightly adjusts the ahrs.roll_trim and ahrs.pitch_trim towards the current stick positions
 // meant to be called continuously while the pilot attempts to keep the copter level
+void Copter::auto_trim_cancel()
+{
+    auto_trim_counter = 0;
+    AP_Notify::flags.save_trim = false;
+    gcs().send_text(MAV_SEVERITY_INFO, "AutoTrim cancelled");
+}
```

Comment:
need a little space here between the methods.

Reply:
Done.

Reply:
no need to add an "auto_trim_started = false" into the cancel right?

Reply:
I did consider doing that!  Then I thought it's reset in the only place that we set auto_trim_counter to a non-negative value, so that was all good.

If time were to be invested you'd create an `AutoTrim auto_trim;` object in Copter, and have an "activate" method on it, called from Motors (and possibly from an RC Aux Switch).  That would hide the magic counter value, group the state and methods, centralize that requirement on being in stabilize (and this make it easier to allow for AltHold), ....